### ISSUE TITLE
Reading WCS keywords

### ISSUE THREAD

**Issue by rwesson** [rwesson/ALFA#5]
Only linearly sampled spectra can be read in at the moment, but the fitting routine is indifferent to the sampling so supporting other cases would just be a matter of reading the relevant FITS keywords.

**Comment by weiaip** [comment]
At least for log-sampled spectra this now (as of v1.0-9-g2fc12c1) works, thanks!
(Although I am not sure what "warning: uncertainty estimation is not reliable for log-sampled spectra" really tells me. I guess I have to re-read the paper to understand what's going on...)

**Comment by rwesson** [comment]
I have fixed the uncertainty calculation for non-linearly sampled spectra, I hope.  ALFA estimates uncertainties using a relation from Lenz & Ayres (1992), which depends on the wavelength sampling.  Previously, that was determined from the wavelength interval of the first two values in the spectrum; I have changed this so that it is determined at the wavelength of the line.  I don't have any non-linearly sampled spectra handy to test this on so would welcome any reports of problems or unrealistic uncertainties!

334e058f1fd804ec4ac6d937b1ecb7c3e3e208c6
